virtumall/settings/base.py

```diff
@@ -99,7 +97,7 @@ def get_env_variable(env_key, default=None):
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
```

Comment:
```suggestion
        "NAME": str(BASE_DIR / "db.sqlite3"),
```

위와 같이 pathlib으로 `BASE_DIR` 설정하고 깔끔하게 관리하는게 낫지 않을까요?